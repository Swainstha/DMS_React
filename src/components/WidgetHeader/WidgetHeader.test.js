import testSnapFunction from '../../util/testSnapFunction';
import WidgetHeader from './index'
import React from 'react';
const dummy_data={
  title:"title",
  extra:"extra",
  styleName:"style"
}
testSnapFunction("<WidgetHeader/>", "Snapshot test for WidgetHeader",<WidgetHeader {...dummy_data}/> )
