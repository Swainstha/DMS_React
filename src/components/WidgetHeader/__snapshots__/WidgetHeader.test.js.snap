// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WidgetHeader/> Snapshot test for WidgetHeader 1`] = `ShallowWrapper {}`;
